lib/dockerregistry/inventory.go

```diff
@@ -1316,32 +1315,18 @@ func mkPopulateRequestsForPromotion(
 				}
 			}
 
-			// Either delete extraneous tags in the destination, or warn about
-			// them (hinges on sc.DeleteExtraTags).
-			mfestIT := mfest.ToRegInvImageTag()
-			if sc.DeleteExtraTags {
-				sc.mkPopReq(
-					registry,
-					destIT.Minus(mfestIT),
-					Delete,
-					"",
-					mkProducer,
-					reqs,
-					wg)
-			} else {
-				// Warn the user about extra tags:
-				xtras := make([]string, 0)
-				for imageTag := range destIT.Minus(promotionCandidatesITRenamed) {
-					xtras = append(xtras, fmt.Sprintf(
-						"%s/%s:%s",
-						registry.Name,
-						imageTag.ImageName,
-						imageTag.Tag))
-				}
-				sort.Strings(xtras)
-				for _, img := range xtras {
-					klog.Warningf("Warning: extra tag found in dest: %s\n", img)
-				}
+			// Warn the user about extra tags:
```

Comment:
We probably want to make this optional also ... when I'm copying images around using cip for dev purposes, these are noise to me.

Reply:
Ah ok. I will create an issue for this (make extra tag warning optional) instead of addressing it here because I have some changes queued up which touch this same area.

Reply:
Created https://github.com/kubernetes-sigs/k8s-container-image-promoter/issues/82